digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/middleware/DataImportManager.scala

```diff
@@ -4,7 +4,7 @@ import java.io.InputStream
 import fi.liikennevirasto.digiroad2.asset._
 import fi.liikennevirasto.digiroad2.service.RoadLinkService
 import fi.liikennevirasto.digiroad2._
-import fi.liikennevirasto.digiroad2.client.vvh.VVHClient
+import fi.liikennevirasto.digiroad2.client.vvh.RoadLinkClient
```

Comment:
Kansio rakenteessa näyttäisi olevan vielä vvh nimi mukana. Pitäisikö myös kansio uudelleen nimetä geneerisemmin?

Reply:
Hyym voi ehkä tullu merge ongelmia, Varmaan ihan viimeisenä vois nimetä johonkin muuhun

Reply:
Jep, kannattaa ehkä katsoa tämä myöhemmin kuntoon. Voisi varmaan kirjata oman tiketin tästä.

Reply:
https://github.community/t/renaming-folder-within-a-repo-loses-file-history/1752/6
https://stackoverflow.com/questions/2314652/is-it-possible-to-move-rename-files-in-git-and-maintain-their-history